src/colorizer/ColorizeCanvas.ts

```diff
@@ -248,8 +266,24 @@ export default class ColorizeCanvas {
     this.setUniform("backgroundColor", color);
   }
 
-  setOutlierColor(color: Color): void {
-    this.setUniform("outlierColor", color);
+  setHideValuesOutOfRange(hide: boolean): void {
+    this.hideValuesOutOfRange = hide;
+    this.setUniform("hideOutOfRange", this.hideValuesOutOfRange);
```

Comment:
this whole function looks like it could be deleted?  all the functionality is in setOutOfRangeDrawMode

Reply:
Oops, I was not as careful during my cleanup pass as I thought I was apparently 😅